test/parser/parse.test.ts

```diff
@@ -351,6 +352,91 @@ See more help with --help`)
         })
         expect(out.flags).to.deep.include({foo: ['a', 'b']})
       })
+      it('allowed options on multiple', async () => {
+        const out = await parse(['--foo', 'a', '--foo=b'], {
+          flags: {
+            foo: Flags.custom({multiple: true, parse: async i => i, options: ['a', 'b']})(),
```

Comment:
Any particular reason for using `Flag.custom` and defining a `parse`? It's not an issue, it just would be more concise to do this:

```
Flags.string({multiple: true, options: ['a', 'b']})
```